### PR TITLE
Telegram Contact message

### PR DESCRIPTION
**Pull request by IvanJov**
Added Telegram Contact message, might be useful 😄

### CODE REVIEW

File: docs/TELEGRAM_CUSTOM_MESSAGES.md [hunk 1]
```diff
@@ -352,6 +352,41 @@ module.exports = botBuilder(message => {
 
 
 
+## Contact messages
+
+Contact template allows you to send a contact with phone number, first name and last name.
```
Comment (simalexan) [inline docs/TELEGRAM_CUSTOM_MESSAGES.md]: @IvanJov could you rephrase the last words here.
`...allows you to share a contact's first name, last name and phone number`

Comment (IvanJov) [inline docs/TELEGRAM_CUSTOM_MESSAGES.md]: Done, thanks for suggestion!

**Comment by simalexan** [comment]
@IvanJov thanks 👍

**Comment by IvanJov** [comment]
@simalexan Any time 👍
